db/schema.rb

```diff
@@ -41,6 +41,16 @@
     t.index ["course_id"], name: "index_assignments_on_course_id"
   end
 
+  create_table "codereview_matchings", force: :cascade do |t|
```

Comment:
We should probably land your PR first anyway. I need npm for the steno websocket integration.